include/openenclave/host.h

```diff
@@ -317,6 +317,24 @@ void oe_free_key(
     uint8_t* key_info,
     size_t key_info_size);
 
+/**
+ * Start the switchless call manager for the enclave. This should be done
```

Comment:
This is not a useful description, since the term "switchless" is not defined and there is no pointer to any doc for more information, so the app developer has no idea what this is for (I have no clue myself).